### ISSUE TITLE
Theano is going away

### ISSUE THREAD

**Issue by kgizdov** [arogozhnikov/hep_ml#67]
Hi,

As per [the statement of MILA](https://groups.google.com/g/theano-users/c/7Poq8BZutbY/m/rNCIfvAEAwAJ) and [Theano page](https://github.com/Theano/Theano) has been going away for a while. I wonder if it has been tried to move towards the currently maintained [Aesara](https://github.com/pymc-devs/aesara). Any thoughts?

**Comment by arogozhnikov** [comment]
Yeah, official support by MILA was ended a while ago, but I am not aware about anything being broken in theano (they also voluntarily do make some adjustments for newer CUDA releases). 

I did not try aesara, and unlikely will have time for that. If you want to migrate hep_ml to aesara - you're welcome to do so!
